### ISSUE TITLE
[CoE Starter Kit - Feature]: FEATURE TITLE

### ISSUE THREAD

**Issue by nickmac88** [microsoft/coe-starter-kit#6535]
### Is your feature request related to a problem? Please describe.

It is not related to a probelm.

### Describe the solution you'd like

We have a scenario where we have some high level organizational people who are makers. I think it is good for these people to still be a part of the standard Maker compliance communications and emails for their specific unused apps (if they have any). However, what we don't want is for these people or the people above them to receive the non-compliance alerts for their subordinates. Given that these folks are VIP's at the company we do not want them receiving these types of notification. 

It might be good if there was a list we could maintain of VIP's (or some other list name) of people never to send these types of communications to. 

We could always force this type of rule through global outlook rules ect but might be good to have this feature built right into the COE.

**Comment by Jenefer-Monroe** [comment]
We do have on our backlog the notion of exempting makers from compliance and archival. But not to exempt them from just manager communications. Its an interesting idea but I'm not sure how that would work. Let me think on that some, I'll put it with that same FC to consider

**Comment by nickmac88** [comment]
Yeah, I think it would be difficult to do it based on an organizational chart. I was thinking more of a list that you could maintain manually. Could be a SharePoint list, dataverse table ect... But if you have people that do not want to receive alerts for whatever reason we could put them on that list.

**Comment by harini-2-y** [comment]
This issue is being closed as its scope is now included in a newly created consolidated enhancement request for centralized management of orphaned components (Apps, Flows, Connection References) in the CoE Starter Kit.

New Issue Reference: [[CoE Starter Kit - Feature]: https://github.com/microsoft/coe-starter-kit/issues/10329

Thank you for your contributions
